scalafmt-core/shared/src/main/scala/org/scalafmt/util/TreeOps.scala

```diff
@@ -856,7 +856,9 @@ object TreeOps {
     }
 
   def getStartOfTemplateBody(template: Template): Option[Token] =
-    template.self.tokens.headOption
+    template.early.headOption
```

Comment:
this function must return the token which starts the body of the class/trait/object, not before that